### ISSUE TITLE
Accessibility bug - keyboard navigation on 'Component' DeclarativeTableCellValue 

### ISSUE THREAD

**Issue by bharathpalaksha** [microsoft/azuredatastudio#25436]
- Azure Data Studio Version: 1.47.0 (user setup)
- Operating System: Windows 11 23H2
- OS Build: 22631.2715
- Extension: Azure Cosmos DB Migration for MongoDB v2.0.3

This accessibility issue is on extension but the ADS framework has provided the UI elements which has the issue.
Flexcontainer within a DeclarativeTable is unable to be accessed by Keyboard navigation, hence the title of image is not being displayed when using keyboard navigation.

Steps to Reproduce:

1. Open Azure data studio application.
2. Tab till Extensions.
3. Serach for Azure cosmos DB migration for MongoDB extension and install.
4. Tab till connections and click on new connection.
5. Select mongo account login with valid credentials.
6. Press alt+Shift+Fn+F10 keys on the connection and press down arrow keys navigate till manage.
7. Tab till azure cosmos DB migration.
8. Tab till Dashboard tab.
9. Tab till New migration button.
10. Tab till Run a new assessment of your MongoDB server.
11. Fill all the details.
12. Tab till run validation and run the validation.
13. Succuss message will appear.
14. Tab till start "Start assessment button".
15. View assessment results and select database (s) for migration will appear.
16. Tab till data base and select any data base.
17. Specify Azure cosmos DB for MongoDB target Account will appear.
18. Fill all the details and click next button.
19. Migration mapping screen will appear.
20. Tab till Target name space.
21. Tab till waring icon.
22. Observed that Warning tooltips is not getting access while using keyboard.

Actual Result:
Warning tooltips is not getting access while using keyboard.
Expect Result:
Warning tooltips should get access while using keyboard.

**Comment by kisantia** [comment]
@bharathpalaksha there have been similar issues about tooltips not showing when navigating with keyboard and they have been downgraded to Sev 3. Please see https://github.com/microsoft/azuredatastudio/issues/22677#issuecomment-1537287812 as an example
